cassandra/src/main/scala/org/apache/zeppelin/cassandra/InterpreterLogic.scala

```diff
@@ -0,0 +1,410 @@
+ * limitations under the License.
+ */
+package org.apache.zeppelin.cassandra
+
+import java.io.{ByteArrayOutputStream, PrintStream}
+import java.net.InetAddress
+import java.nio.ByteBuffer
+import java.text.SimpleDateFormat
+import java.util
+import java.util.Date
+import java.util.concurrent.ConcurrentHashMap
+
+import com.datastax.driver.core.DataType.Name._
+import com.datastax.driver.core._
+import com.datastax.driver.core.exceptions.DriverException
+import com.datastax.driver.core.policies.{LoggingRetryPolicy, FallthroughRetryPolicy, DowngradingConsistencyRetryPolicy, Policies}
+import org.apache.zeppelin.cassandra.TextBlockHierarchy._
+import org.apache.zeppelin.display.Input.ParamOption
+import org.apache.zeppelin.interpreter.InterpreterResult.Code
+import org.apache.zeppelin.interpreter.{InterpreterException, InterpreterResult, InterpreterContext}
+import org.slf4j.LoggerFactory
+import scala.collection.JavaConversions._
+import scala.collection.JavaConverters._
+import scala.collection.mutable
+import scala.collection.mutable.ArrayBuffer
+
+
+/**
+ * Value object to store runtime query parameters
+ * @param consistency consistency level
+ * @param serialConsistency serial consistency level
+ * @param timestamp timestamp
+ * @param retryPolicy retry policy
+ * @param fetchSize query fetch size
+ */
+case class CassandraQueryOptions(consistency: Option[ConsistencyLevel],
+                                 serialConsistency:Option[ConsistencyLevel],
+                                 timestamp: Option[Long],
+                                 retryPolicy: Option[RetryPolicy],
+                                 fetchSize: Option[Int])
+
+/**
+ * Singleton object to store constants
+ */
+object InterpreterLogic {
+  
+  val CHOICES_SEPARATOR : String = """\|"""
+  val VARIABLE_PATTERN = """\{\{[^}]+\}\}""".r
+  val SIMPLE_VARIABLE_DEFINITION_PATTERN = """\{\{([^=]+)=([^=]+)\}\}""".r
+  val MULTIPLE_CHOICES_VARIABLE_DEFINITION_PATTERN = """\{\{([^=]+)=((?:[^=]+\|)+[^|]+)\}\}""".r
+
+  val STANDARD_DATE_FORMAT = "yyyy-MM-dd HH:mm:ss"
+  val ACCURATE_DATE_FORMAT = "yyyy-MM-dd HH:mm:ss.SSS"
+
+  val defaultRetryPolicy = Policies.defaultRetryPolicy()
+  val downgradingConsistencyRetryPolicy = DowngradingConsistencyRetryPolicy.INSTANCE
+  val fallThroughRetryPolicy = FallthroughRetryPolicy.INSTANCE
+  val loggingDefaultRetryPolicy = new LoggingRetryPolicy(defaultRetryPolicy)
+  val loggingDownGradingRetryPolicy = new LoggingRetryPolicy(downgradingConsistencyRetryPolicy)
+  val loggingFallThrougRetryPolicy = new LoggingRetryPolicy(fallThroughRetryPolicy)
+
+  val preparedStatements : mutable.Map[String,PreparedStatement] = new ConcurrentHashMap[String,PreparedStatement]().asScala
+
+  val logger = LoggerFactory.getLogger(classOf[InterpreterLogic])
+
+  val paragraphParser = new ParagraphParser
+  val boundValuesParser = new BoundValuesParser
+
+  def toScalaList[A](list: java.util.List[A]): List[A] = list.asScala.toList
```

Comment:
here you go 
```
    public <A> scala.collection.immutable.List<A> toScalaList(java.util.List<A> list)  {
        return scala.collection.JavaConverters.asScalaIterableConverter(list).asScala().toList();
    } 
    
    public <A> java.util.List<A> toJavaLis(scala.collection.immutable.List<A> list){
       return scala.collection.JavaConverters.asJavaListConverter(list).asJava();
    }
```

or create a helper class in the scala test folder with the conversion methods but don't keep them in production code